### ISSUE TITLE
0.1.9: sphinx 4.2 warnings

### ISSUE THREAD

**Issue by kloczek** [sphinx-contrib/emojicodes#33]
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
running build_sphinx
Running Sphinx v4.2.0
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [man]: all manpages
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index
/home/tkloczko/rpmbuild/BUILD/emojicodes-0.1.9/docs/source/index.rst:5: WARNING: Inline emphasis start-string without end-string.
/home/tkloczko/rpmbuild/BUILD/emojicodes-0.1.9/docs/source/index.rst:5: WARNING: Inline emphasis start-string without end-string.
/home/tkloczko/rpmbuild/BUILD/emojicodes-0.1.9/docs/source/index.rst:5: WARNING: Inline emphasis start-string without end-string.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... sphinxemojicodes.1 { } done
build succeeded, 3 warnings.
```
BTW: as license in setup.py is listed `license='BSD License'` howevewrthere are few BSD licenses:
BSD-2-Clause (https://opensource.org/licenses/BSD-2-Clause/)
BSD-3-Clause (https://opensource.org/licenses/BSD-3-Clause/)
BSD-4-Clause (https://spdx.org/licenses/BSD-4-Clause.html)

May I ask for claryfication which one in thi case is used?

**Comment by Peque** [comment]
@kloczek BSD 3. :blush:

**Comment by kloczek** [comment]
Thx :)

**Comment by Peque** [comment]
Thanks to you! The file should have been added earlier... :sweat_smile: 

I just merged the license file: https://github.com/sphinx-contrib/emojicodes/pull/34

Feel free to contribute a pull request to get rid of those warnings! :blush:

**Comment by Peque** [comment]
@kloczek Fixed with https://github.com/sphinx-contrib/emojicodes/pull/35/files. Changes should be available in the latest version (0.2.0).

Thanks for pushing me to fix this! :blush:
